tests/testthat/test-scatter_plot.R

```diff
@@ -25,6 +33,14 @@ testthat::test_that("Function can only accept certain data types for arguments",
   
 })
 
+#test type of scatter_plot
+testthat::test_that("Function has the right type", {
+  expect_true(rlang::is_empty(type), helper_scatter_empty_type)
```

Comment:
I think you have to specify the function here, e.g. `scatter_plot(mtcars, mpg, hp, "Miles Per Gallon", "Horse Power", "Scatter Plot for MPG vs HP", 10)` instead of `rlang::is_empty(type)`.